### ISSUE TITLE
Feature request: Give a hint if some option need restart to take effect

### ISSUE THREAD

**Issue by xaljer** [variar/klogg#432]
I found that "keep file close" seems need restart to take effect. If some options do need restart, it's better to give a hint to user.

**Comment by variar** [comment]
Yes, that's inconvinient. However, that should work when file is closed and reopened.  Some "follow" options also work that way. It would be better to find a way of applying them on-the-fly.

**Comment by xaljer** [comment]
Maybe we can done this by two steps: first, give a description around the option, and then try to apply them on-the-fly.
